### PR TITLE
Added demo for blog, more instructions

### PR DESCRIPTION
**Pull request by harowo**

### CODE REVIEW

File: News App/README.MD [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # This is the folder for News App templates
 
 Open a Pull Request following the naming convention `News App/description_of_PR`
+
+## For a demo of submission, refer to [Food Blog Demo](https://github.com/dunmanhigh/web_templates/blob/main/Blog/Demo%20Food%20Blog)
```
Comment (joelleoqiyi) [inline News App/README.MD:5]: what is this for?

File: Blog/Demo Food Blog/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# This is a demo of your submission
+
+## Files included
+
+- HTML file ([food_blog_demo.md](https://github.com/dunmanhigh/web_templates/blob/main/Blog/Demo%20Food%20Blog/food_blog_demo.html))
+- Image of your sketch ([food_blog_demo_sketch.jpg](https://github.com/dunmanhigh/web_templates/blob/main/Blog/Demo%20Food%20Blog/food_blog_demo_sketch.jpg_))
+- Optional: Amended HTML file ([food_blog_demo_amended.md](https://github.com/dunmanhigh/web_templates/blob/main/Blog/Demo%20Food%20Blog/food_blog_demo_amended.html))
```
Comment (joelleoqiyi) [inline Blog/Demo Food Blog/README.md]: `.html` instead?

File: Blog/Demo Food Blog/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# This is a demo of your submission
+
+## Files included
+
+- HTML file ([food_blog_demo.md](https://github.com/dunmanhigh/web_templates/blob/main/Blog/Demo%20Food%20Blog/food_blog_demo.html))
```
Comment (joelleoqiyi) [inline Blog/Demo Food Blog/README.md]: `.html` instead?

**Review by joelleoqiyi — COMMENTED** [review]
not sure if we need to add in the demo link for other than blog wdyt?

File: News App/README.MD (hunk 1 above)
Comment (harowo) [reply News App/README.MD:5]: demo folder is for what is required in submission, not only applicable to blog

File: Blog/Demo Food Blog/README.md (hunk 2 above)
Comment (harowo) [reply Blog/Demo Food Blog/README.md]: yes

**Comment by joelleoqiyi** [comment]
LGTM.

**Pull request merged**